tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs

```diff
@@ -1,19 +1,17 @@
 namespace FakeItEasy.IntegrationTests
 {
     using System;
-    using System.Globalization;
 
     public static class StringArgumentConstraints
     {
         public static string StartsWith(this IArgumentConstraintManager<string> scope, string beginning)
         {
-            return scope.Matches(
-                x => x.StartsWith(beginning, StringComparison.CurrentCulture), string.Format(CultureInfo.CurrentCulture, "Starts with \"{0}\"", beginning));
+            return scope.Matches(x => x.StartsWith(beginning, StringComparison.CurrentCulture), $@"Starts with ""{beginning}""");

```

Comment:
I considered, but it was only the one instance that was giving me grief, and string _comparison_ is a different beast than _formatting_, so I shied away.
I'd make a separate PR if we wanted to pursue this…